src/main/java/frc/robot/constants/BlankConstants.java

```diff
@@ -0,0 +1,117 @@
+package frc.robot.constants;
+
+import frc.robot.drive.MotorType;
+
+public class BlankConstants implements Constants {
```

Comment:
We should add a class constant describing the purpose of this object. Specifically, that it has no motor controllers, no sensors, etc.